examples/update-organization.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Could you rename it to organization-update.js ? Other example files follow the `entity-activity` format

Reply:
Updated.  👍